.storybook-s2/docs/Migrating.jsx

```diff
@@ -0,0 +1,770 @@
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>[PENDING] Comment out <Code>trigger</Code> (it has not been implemented yet)</li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Update <Code>Item</Code> to be a <Code>MenuItem</Code></li>
+        </ul>
+
+        <H3>Avatar</H3>
+        <ul className="sb-unstyled">
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>[PENDING] Comment out <Code>isDisabled</Code> (it has not been implemented yet)</li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Remove <Code>size</Code> and instead provide a size via the style macro (i.e. <Code>{`styles={style({size: 20})}`}</Code>)</li>
+        </ul>
+
+        <H3>Breadcrumbs</H3>
+        <ul className="sb-unstyled">
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>[PENDING] Comment out <Code>showRoot</Code> (it has not been implemented yet)</li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>[PENDING] Comment out <Code>isMultiline</Code> (it has not been implemented yet)</li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>[PENDING] Comment out <Code>autoFocusCurrent</Code> (it has not been implemented yet)</li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Remove <Code>size="S"</Code> (Small is no longer a supported size in Spectrum 2)</li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Update <Code>Item</Code> to be a <Code>Breadcrumb</Code></li>
+        </ul>
+
+        <H3>Button</H3>
+        <ul className="sb-unstyled">
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Change <Code>variant=“cta”</Code> to <Code>variant="accent"</Code></li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Change <Code>variant=“overBackground”</Code> to <Code>variant=“primary” staticColor=“white”</Code></li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Change <Code>style</Code> to <Code>fillStyle</Code></li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>[PENDING] Comment out <Code>isPending</Code> (it has not been implemented yet)</li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Remove <Code>isQuiet</Code> (it is no longer supported in Spectrum 2)</li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>If <Code>href</Code> is present, the <Code>Button</Code> should be converted to a <Code>LinkButton</Code></li>
+          <li className={style({lineHeight: 'body', color: 'body', marginY: 8})}>Remove <Code>elementType</Code> (it is no longer supported in Spectrum 2)</li>
+        </ul>
+
+        <H3>ButtonGroup</H3>
```

Comment:
Mega nit: some of these don't say "No updates needed". Is it because they will have updates later?

Reply:
These should also say "No updates needed". Thanks.